pkg/devspace/services/synccontroller/controller.go

```diff
@@ -226,10 +233,12 @@ func (c *controller) startSync(options *Options, onInitUploadDone chan struct{},
 	)
 
 	localPath := "."
+	if runtime.GOOS == "darwin" {
```

Comment:
Can you put the runtime.GOOS check inside the function?

Reply:
done